Src/BootCamp.Chapter/Lesson10.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson10
+    {
+        public static string EncryptCypher(string message, int shift)
+        {
+            if (shift < 0)
+            {
+                shift = Math.Abs(shift + 95);
+            }
+
+            var messageBytes = Encoding.ASCII.GetBytes(message);
+
+            for (int i = 0; i < message.Length; i++)
+            {
+                var currentCharByte = messageBytes[i];
+                byte newByte = Convert.ToByte((((currentCharByte - 32) + shift) % 95) + 32);
+                messageBytes[i] = newByte;
+            }
+
+            var newMessage = Encoding.ASCII.GetString(messageBytes);
+
+            return newMessage;
+
+        }
+
+        public static string DecryptCypher(string message, int shift)
```

Comment:
looks like you are encrypting an already encrypted message again to decrypt it, it works because the way you used but you should make a general function and use it in both Decrypt and Encrypt methods.

Reply:
Cypher function created that both use.